chart/templates/webserver/webserver-deployment.yaml

```diff
@@ -94,7 +94,7 @@ spec:
         - name: webserver
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
-          args: ["webserver"]
+          args: ["airflow", "webserver"]
```

Comment:
Should it be also adjusted in workers deployment?

Reply:
@turbaszek the worker deployments already use airflow commands

Reply:
https://github.com/astronomer/airflow/blob/fix-entrypoint-for-openshift/chart/templates/workers/worker-deployment.yaml#L118

Reply:
Oh indeed, good!